documentation/book/con-kafka-listeners.adoc

```diff
@@ -123,4 +123,48 @@ listeners:
 ----
 
 
-Authentication must be configured when using the User Operator to manage `KafkaUsers`.
\ No newline at end of file
+Authentication must be configured when using the User Operator to manage `KafkaUsers`.
+
+== Network Policies
+
+For every enabled listener, {ProductName} will automatically create a NetworkPolicy which will by default grant all applications access to the listener.
+You can restrict the access only to selected applications using the `networkPolicyPeers` field.
+You can use different `networkPolicyPeers` configuration for each listener.
+
+.An example where the `plain` and `tls` listeners allow connections only from selected pods
```

Comment:
_Insert the following paragraph instead of the sub-heading:_

The following example shows a `networkPolicyPeers` configuration for a `plain` and a `tls` listener: